nextflow_schema.json

```diff
@@ -4,8 +4,9 @@
     "title": "nf-core/atacseq pipeline parameters",
     "description": "ATACSeq peak-calling and differential analysis pipeline.",
```

Comment:
```suggestion
    "title": "nf-core/chipseq pipeline parameters",
    "description": "ChIP-seq peak-calling and differential analysis pipeline.",
```

Reply:
Sorry, this was my bad for copying the schema from `atacseq` and not checking properly. Have updated in `dev` but makes sense to do the same here too. The `$id` line above needs updating too.

Reply:
Hah, I was literally editing the file to fix this as your comment coming in..

Reply:
Great minds mate! 🤓